test(rsc): fail fast when fixture server exits (#1367)
vitejs/vite-plugin-react: packages/plugin-rsc/AGENTS.md

## Quick Reference for AI Agents

### Fresh Worktree Setup

```bash
# from the repository root
pnpm install
pnpm build
```

> Run the full workspace build before RSC E2E tests because examples may import other workspace packages, including `@vitejs/plugin-react`.

Author: ```suggestion
Run the full workspace build before RSC E2E tests because examples use other workspace packages, such as `@vitejs/plugin-react`.
```
